src/pages/workspace/WorkspaceInitialPage.js

```diff
@@ -35,6 +35,21 @@ const propTypes = {
 
     /** The employee list of this policy (coming from Onyx) */
     policyMemberList: PropTypes.objectOf(policyMemberPropType),
+
+    /** All reports shared with the user (coming from Onyx) */
+    reports: PropTypes.shape({
+        /** The report name */
+        reportID: PropTypes.number,
```

Comment:
This just changed to `string` everywhere: https://github.com/Expensify/App/pull/11362

```suggestion
        reportID: PropTypes.string,
```

Reply:
Also, this propType doesn't seem right. Should it be:

```js
reports: PropTypes.objectOf(reportPropTypes),
```

Where `reportPropTypes` comes from [this new file on main](https://github.com/Expensify/App/blob/main/src/pages/reportPropTypes.js)?

Reply:
Updated